android/src/main/java/sqip/react/CardEntryModule.java

```diff
@@ -301,7 +301,7 @@ private BuyerAction getBuyerAction(final String buyerActionString, final Money m
   private long readCardEntryCloseExitAnimationDurationMs() {
     long delayDurationMs = 0;
     Resources.Theme theme = Objects.requireNonNull(getCurrentActivity()).getResources().newTheme();
-    theme.applyStyle(R.style.sqip_Theme_CardEntry, true);
+    theme.applyStyle(R.style.sqip_ThemeCardEntry, true);
```

Comment:
@WhatsEmo, am I right that this smells bad?

@mansi-square, I believe we tell users they can override the (previous) theme values to style their apps, so changing to an almost-same name and defining a new style below is potentially breaking to people who are customizing the styles.  Are we gaining anything by this rename?

Reply:
> am I right that this smells bad?

Yep, I think this would crash or not reference a theme at all since `R.style.sqip_Theme_CardEntry` is the resource value we ask developers to use.

Reply:
Ahh it wouldn't crash because <style name="sqip_ThemeCardEntry" parent="@style/Theme.AppCompat.DayNight.DarkActionBar" /> was added, but since the theme is set to a default Android style it wouldn't know what values to override when the develop follows https://developer.squareup.com/docs/in-app-payments-sdk/cookbook/customize-payment-form

Reply:
- [ ] Revert this, or find a way to update that allows the customization from our documented theming.

Reply:
With the new version of React-Native some modules are removed so the default style which android was providing is now removed and regarding to change in name even i was giving same name to the new style it was throwing duplicate name error.